library/koji_tag_inheritance.py

```diff
@@ -144,7 +144,13 @@ def add_tag_inheritance(session, child_tag, parent_tag, priority, check_mode):
             # Mark this rule for deletion
             delete_rule['delete link'] = True
             new_rules.insert(0, delete_rule)
-    result['stdout'] = 'set parent %s (%d)' % (parent_tag, priority)
+            result['stdout_lines'].append(
```

Comment:
I'm tempted to drop the "would" code here, because it's kind of hard to read, and just log "remove foo" or "set foo". Then we have the same messages in check mode or non-check mode. What do you think?

Reply:
Good idea; done.